ev3dev/core.py

```diff
@@ -909,6 +909,8 @@ def wait(self, cond, timeout=None):
             if cond(self.state):
                 return True
 
+    def wait_until_not_moving(self, timeout=None):
```

Comment:
You should add a documentation comment here so that it shows up on the docs site.